### DISCUSSION TITLE
Amiga diskspare format

### DISCUSSION THREAD

**Discussion by NebularNerd** [keirf/greaseweazle#161] [Q&A] [answered]
Hi everyone,

I thinking of getting a greaseweazle to backup my hoard of floppies most of which are Amiga flavoured. As Amiga fans will know there was a time when coverdisks were supplied in [Diskspare format](http://aminet.net/package/disk/misc/Diskspr3) and on top of that I used the format for some of my personal disks (because space was precious back then)

I have made some ropey grade backups in the dim distant past with .dms images and such but would really like to grab the disks in the native form. Does anyone know if greaseweazle can grab the diskspare format and make images usable in UAE?

Also has any tried grabbing those weird 50/50 format disks with Amiga on one side and ST/PC on the other?

Thanks for any help 🙂

**Comment by M1kerochip** [comment, marked as answer]
Yes, and both things work fine, from the ones I've made. Coverdisks imaged from Zero magazine that were Amiga/ST/PC and several commercial games, and Amiga User International, that used the above extended disk format worked fine for me.

It depends on the quality of the disk you're backing up, though, as to if you'll get a working disk image.

DMS as a format though, had flaws in it I thought, which could create disk errors. Not sure if that was ever fixed or not, so I never used DMS.

**Comment by M1kerochip** [comment]
Here's some Zero disks I and others have made, if you want to look at some. https://mega.nz/folder/9RJixYhI#ozJAY06nUpYj2vzpuWORRg/folder/8doVyQIC
Issue 32 is Amiga/ST/PC btw!

**Comment by NebularNerd** [comment]
Hi @M1kerochip,

Thanks for the info 🙂 I used the DMS format waaaaaayyyyyy back when there was almost no other way to image a disk, unless you could afford a Catweasel interface back then. Those things were rarer than rocking horse droppings, especially after the German legislation changes outlawed half the components they were made from. I guess the old DMS's may be handy in case any of the disks have not survived since that time but this is partly why I wish to grab everything I have on floppy media in a modern archive format (or at least .spc) 

I'll check out your link as well when I have some more free time 🙂
